api.md

```diff
@@ -0,0 +1,86 @@
+# Proposed webrtc webextension api
+
+The initial version of the webrtc webextension api is a limited
+set of events, just enough to implement the extension from
+[this mozhacks article](https://hacks.mozilla.org/2015/09/controlling-webrtc-peerconnections-with-an-extension/).
+
+To use any of these events, an extension must have the `"webrtc"`
+permission (in addition to the `"experiments.webrtc"` permission unless/until
+this api lands in central).  In addition, an extension must have the
+`"webrtc.blocking"` permission to add a blocking onPeerConnection listener.
+
+The events are:
+
+## onPeerConnection
```

Comment:
Good question, I'm even less familiar with the ICE protocol than with the webrtc apis...
Maybe something like onIceAddressCandidate?  Ugh.

For the peer connection events, how about `onPeerConnectionRequest` and `onPeerConnectionRequestCanceled`?  Again, I think that making a permission decision is the common application for these events but but I could also see somebody using them to do something like create a log of all peer connections or create some extra visual indicator of a connection, so embedding the word permission in the event name doesn't feel right.